data/tiles.yml

```diff
@@ -2,12 +2,13 @@
 test_analytics_features:
   - title: "Deep performance analysis"
     url: "/docs/test-analytics/test-suites#trends-and-analysis"
-    image_path: "images/test_analytics/icon-performance-analysis.svg"
```

Comment:
should these images have been removed 🤔

Reply:
Yup, I mentioned in the description that the overview icons are being removed [from the request here ](https://linear.app/buildkite/issue/PIE-1974/add-issues-page-to-docs#comment-9a14d5f5)